### ISSUE TITLE
sulu_content_path does not consider hostname 

### ISSUE THREAD

**Issue by afroewis** [sulu/sulu-standard#787]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.4.3

My client reported this to me after upgrading from 1.3.0 to 1.4.3 and I just reproduced it and had a more detailed look at it. 

In our project, every locale has its own URL: 

```
<environment type="prod">
                    <urls>
                        <url language="de">www.beispiel.org</url>
                        <url language="en">www.example.org</url>
                        <url language="es">www.ejemplo.org</url>
                        ...
                    </urls>
</environment>
```

Let's say we have a page translated in `de` and `en`. Their absolute URLs would be `www.beispiel.org/seite-1` and `www.example.org/page-1`.

On our site, we have a language switch, so the user can change the language of the current page if it is translated. This is how we do it (simplified): 

```
{% set availableLocales = sulu_content_load(uuid).urls %}
<ul class="dropdown-menu">
                {% for key, language in locales %}
                        <li><a href="{{ sulu_content_path(availableLocales[key], null, key) }}">{{ language }}</a></li>
                {% endfor %}
</ul>
```

So (if you simplify it even more), this would be the same as:
 
```
<a href="{{ sulu_content_path('/seite-1', null, 'de') }}">
<a href="{{ sulu_content_path('/page-1', null, 'en') }}">
<a href="{{ sulu_content_path('/pagina-1', null, 'es') }}">
```
### Expected behavior

In 1.3.0 (and as the docs state), this would render as: 

```
<a href="www.beispiel.org/seite-1">
<a href="www.example.org/page-1">
<a href="www.ejemplo.org/pagina-1">
```
### Actual behavior

But in 1.4.3 it renders as:
```
<a href="www.beispiel.org/seite-1">
<a href="www.beispiel.org/page-1">
<a href="www.beispiel.org/pagina-1">
```

So it does not consider that the locales have different hostnames. Obviously, those URLs do not exist. So currently if a user changes the language, a 404 error is shown.

**Comment by danrot** [comment]
@afroewis Can reproduce, I am having a look at it.

**Comment by danrot** [comment]
Fixed in sulu/sulu#3131
